lib/auth/session_access.go

```diff
@@ -36,17 +37,28 @@ import (
 // that is harder to debug in the case of misconfigured policies or other error and are harder to intuitively follow.
 // In the real world, the number of roles and session are small enough that this doesn't have a meaningful impact.
 type SessionAccessEvaluator struct {
-	kind       types.SessionKind
-	policySets []*types.SessionTrackerPolicySet
+	kind        types.SessionKind
+	policySets  []*types.SessionTrackerPolicySet
+	IsModerated bool
```

Comment:
Should this be exported? Do external clients ever need to set this, or does the access evaluator determine on its own when the session is moderated?

Reply:
We never need to set this but we do read it. Switched to a getter to prevent accidental assignment